proto/Networking/Requests/Messages/PlatformClientActionsMessage.proto

```diff
@@ -38,24 +39,63 @@ message RedeemPasscodeMessage {
 }
 
 message AddNewPoiMessage {
+	string title = 1;
+	string long_description = 2;
+	string image_gs_file_path = 3;
+	int32 lat_e6 = 4;
+	int32 lng_e6 = 5;
+	string image_serving_url = 6;
+	string user_id = 7;
+	string player_language = 8;
+	string game_unique_id = 9;
+	string app_id = 10;
+	.POGOProtos.Data.Player.PlayerReputation player_reputation = 11;
+}
+
+message ProxySocialSideChannelActionMessage {
+	uint32 action = 1;
+	string host = 2;
+	bytes payload = 3;
+}
+
+message GetAvailableSubmissionsMessage {
+}
+
+message PushNotificationRegistryMessage {
+	RegisterPushNotificationMessage.ApnToken apn_token = 1;
+	RegisterPushNotificationMessage.GcmToken gcm_token = 2;
+}
+
+message AddLoginActionMessage {
+	.POGOProtos.Enums.IdentityProvider identity_provider = 1;
+	bytes inner_message = 2;
+}
+
+message RemoveLoginActionMessage {
+	.POGOProtos.Enums.IdentityProvider identity_provider = 1;
+}
+
+message SubmitNewPoiMessage {
 	string title = 1;
 	string long_description = 2;
 	int32 lat_e6 = 4;
 	int32 lng_e6 = 5;
 	string supporting_statement = 14;
 }
 
-message ProxySocialActionMessage {
```

Comment:
if you removed it from there, where did you put it?

Reply:
c'est le même que https://github.com/pogosandbox/node-pogo-protos/blob/master/proto/Networking/Responses/PlatformClientActionsResponse.proto#L84  (je sais pas si c'est doublon ou pas)

Reply:
non c'est pas le meme

Reply:
ok ok

Reply:
fixed https://github.com/pogosandbox/node-pogo-protos/pull/19/commits/97be62a1368a48568334942b6d64679d681dd1a7